common/api/summary/itwin-registry-client.exports.csv

```diff
@@ -0,0 +1,9 @@
+sep=;
```

Comment:
Can you delete the current "context-registry-client" files?

Reply:
Should I also delete the folder under common/changes/@bentley?

Reply:
Yes, go ahead.